### DISCUSSION TITLE
It looks like flood can't recognize peer running client Deluge 2.x

### DISCUSSION THREAD

**Discussion by lilws** [jesec/flood#285] [General]
Recently I checked peers tab and find some unknown client in flood, then I open rutorrent and it recognized the client correctly.

This happen in flood
<img width="832" alt="Screen Shot 2021-04-14 at 13 58 46" src="https://user-images.githubusercontent.com/12478478/114667745-0ef42280-9d2a-11eb-96e6-851936292bc5.png">

In rutorrent
<img width="649" alt="Screen Shot 2021-04-14 at 13 59 27" src="https://user-images.githubusercontent.com/12478478/114667791-1e736b80-9d2a-11eb-8dd9-34b46e48cea4.png">

**Comment by jesec** [comment]
Flood uses client name responses from the rTorrent, while ruTorrent parses client IDs itself. Unfortunately, the vanilla rTorrent has outdated client ID mappings. I applied some patches to my distribution, which should help it to properly identify more clients.

**Reply by lilws** [reply]
That's good to know. I hope rtorrent dev will active again sometimes. It has been outdated almost 2 years.
